src/app/editor/components/Toolbar/index.tsx

```
@@ -320,35 +328,34 @@ export default function Toolbar() {
 				</div>
 
 				<div className="flex flex-col gap-y-1.5">
 					<button
-						type="button"
+						className="group flex justify-center items-center gap-2 rounded-lg border border-emerald-400/20 bg-emerald-400/5 px-6 py-1.5 font-semibold text-sm text-emerald-400 uppercase tracking-wide backdrop-blur-sm transition-all duration-300 hover:border-emerald-400/40 hover:bg-emerald-400/10"
 						id="toolbar-import"
-						className="w-full rounded-lg bg-gradient-to-r from-emerald-400 to-cyan-400 py-1 font-semibold text-sm text-white uppercase tracking-[2px]"
 						onClick={() =>
 							(hiddenFileInput.current as HTMLInputElement).click()
 						}
 					>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)